addons/website/views/snippets/s_experience.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_experience" name="Experience">
+    <section class="s_experience pt104 pb104">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="11">
+                <div class="o_grid_item g-height-8 g-col-lg-8 col-lg-6 align-content-center pt96 pb96" data-name="Box" style="z-index: 2; grid-area: 1 / 3 / 10 / 11; text-align: center;">
```

Comment:
Following website's conventions for grid items:
- Order element in the DOM according to the desktop visual result (in this case: image block, text block, image block)
- Adapt `z-index` values according to the element order on desktop
- If something needs to change position on mobile, add `order: [x];` inline style + the `order-lg-0` class